README.md

```diff
@@ -78,12 +78,35 @@ Common errors you may get when running the ```docker-compose build``` or ```dock
 | Error about "apt-get update -qq" failing                                            | Run `curl -sSL https://get.docker.com/ \| sh` and then rerun the ```docker-compose build``` command.                                                                                                                                                |
 | driver failed programming external connectivity on endpoint <IP Address>            | This is a common issue on Windows 10, try stopping all docker containers with ```docker stop $(docker ps -a -q)``` then restart docker on your machine and try again. See [here](https://github.com/docker/for-win/issues/573) for more information |
 | Windows Docker-Compose Error  /usr/bin/env: ‘python\r’: No such file or directory | Open run.py in vi or vim (access through git bash) - `vi run.py`, type `:set ff=unix` and save and edit `:wq`.                                                                                                                                      |
+## Frequently Asked Questions
 
+* **I have never worked on a large scale project like this before. Should I jump right in, or how much code should I look at first?**
 
-## License
+    This depends on you. The most important thing to remember is that you don't have to worry about breaking anything. All code is reviewed and tested before being deployed so jumping right in or taking a slower and more measured approach are both fine. The issue descriptions should also contain enough information to get you started and if not, a comment asking for clarification is more than welcome.
+
+* **What are the expected time frames regarding the high and low priority?**
+
+    High priority: pull request within two days.
+
+    Low priority: pull request after seven to nine days.
+
+* **I can’t help right now, but I should free up in a month or two - can I still help?**
+
+    Baobab will log a continuous stream of new issues, bugs that creep in, or new features needed. 
+
+* **My country doesn’t allow for docker to installed**
 
-This project is licensed under the Apache 2.0 License - see the [LICENSE](LICENSE) file for details.
+    [Windscribe](https://windscribe.com/) is a free VPN that doesn’t require a credit card for set up and can potentially help with the install. Tor browsers are also a potential avenue to try out. If you find a workaround that works, please let us know !
 
-## Acknowledgments
+* **I have a question about the issue, where should I ask?**
+    All questions relating to a specific issue, should be left on the GitHub on the issue itself (or the Pull Request if you've opened one).
+
+    Please be as descriptive as possible, and include any error messages you might be experiencing.
+
+    If you have any time, please peruse the other issues and see if there are other questions you can answer. We are here to learn from each other!
+
```

Comment:
It might be worth stating how much space is needed for docker etc.? Especially if someone is going to need to partition their drive for Ubuntu.

Reply:
Oh yes that would be quite useful to know. I've added it now based on how much space I'm using. 4.7GB.